bazel/foreign_cc/BUILD

```diff
@@ -103,6 +103,16 @@ configure_make(
     tags = ["skip_on_windows"],
 )
 
+configure_make(
+    name = "ragel",
+    env = {
+        "CXXFLAGS": "-lstdc++",
```

Comment:
Does the library statically link libstdc++? e.g. like https://github.com/envoyproxy/envoy/blob/0a4e3d990ae865c6753432322e9b2eaec8057548/bazel/foreign_cc/BUILD#L412 ?

Reply:
Yes, since the output binary is required for compiling Hyperscan. Comment added.

Reply:
Actually, I have not tested building with libc++, but the CI does not complain about it. Does our CI check if it can be built with libc++?

Reply:
I'm actually not sure what build setting we use to build contrib. I suggest trying building with the --config=libc++ just to see what happens.

Reply:
I have run `ENVOY_STDLIB=libc++ ./ci/run_envoy_docker.sh './ci/do_ci.sh bazel.release'` which will build and test envoy-contrib in `envoyproxy/envoy-build-ubuntu` Docker environment with libc++, and the build and tests finished successfully.

Reply:
@zhxie  have you tried delete the linked `-lstdc++`? I have tried it with `libc++` build and the compile OK.
so suppose libstdc++ is not necessary here?